mainlyetcetera/fitlit
# Feature/write user tests

### What does this PR do?
- This PR adds 3 tests for the User class.

### What does it fix?
- No bug being fixed at this time.

### Is this a feature or a fix?
- I would say this is a feature.

### Where should the reviewer start?
- Open User-test.js.

### How should this be tested?
- Inside fit lit directory, run npm test test/User-test.js, and see there are 3 tests pending.
